gulpfile.js

```diff
@@ -26,7 +26,7 @@ var Promise = require('bluebird');
 var requirejs = require('requirejs');
 var Karma = require('karma').Server;
 var yargs = require('yargs');
-var aws = require('aws-sdk');
+var S3 = require('aws-sdk/clients/s3');
```

Comment:
You should never index into a node module like this.  require in `aws-sdk` like before and then use `AWS.S3`